consul/datadog_checks/consul/consul.py

```diff
@@ -77,18 +77,9 @@ def __init__(self, name, init_config, instances):
         self._instance_states = defaultdict(lambda: ConsulCheckInstanceState())
 
         self.HTTP_CONFIG_REMAPPER = {
```

Comment:
All of these were looking into init_config while this config options are instance options. Also I think the defaults as false are true. Certs if not provided are just None and verify should default to True

Reply:
@hithwen Based on the initial code https://github.com/DataDog/integrations-core/pull/3363/files#diff-92c6a37ebfb4a295afacabd06258aff3L84-L86

looking into the init config seems the right thing to do to keep the implementation backward compatible.

Reply:
Original PR for context: https://github.com/DataDog/integrations-core/pull/3363